activities/member-relations/information-about-founding-membership.md

```diff
@@ -0,0 +1,21 @@
+---
+type: Resource
+---
+
+# Information about Founding Membership template
+
+## Resource
+
+You can access the [template here](https://docs.google.com/document/d/165RXZmeU6klU_QmCR06ACvY8SAtN27de1nilqOmi8Nw/edit?usp=sharing).
```

Comment:
This is fine, but for good form (and printing this with relevant styling) you should make the link a description of what it is, e.g.

```suggestion
The template is in [the Google Doc "Template_Information about Founding Membership"](https://docs.google.com/document/d/165RXZmeU6klU_QmCR06ACvY8SAtN27de1nilqOmi8Nw/edit?usp=sharing).
```

Reply:
see #329